server/py/services/api/crud/projects.py

```diff
@@ -203,16 +204,18 @@ def delete_project_resources(
             project_name=name,
         )
 
-        # TODO: Forward to alerts service
-        # The messaging client is async, and project deletion is sync.
-        # When deleting a project, we need to use a sync client to send the delete event request to the alerts service,
-        # or to Chief if in Hydra mode. (ML-8390)
-        # Until we implement the sync client, we can allow Chief to delete the project alerts itself, instead of
-        # actually forwarding the request and waiting for a response, since the project deletion flow is handled
-        # by Chief only.
+        # TODO: This should be refactored once we have a proper hydra implementation
+        # Delete alert's service resources
         services.alerts.crud.Events().delete_project_alert_events(name)
```

Comment:
this can also be part of condition

Reply:
why is this needed? It's done in `services.alerts.crud.Alerts().delete_alerts`
https://github.com/mlrun/mlrun/blob/development/server/py/services/alerts/crud/alerts.py#L191

Reply:
@alonmr you're right, my bad, can be removed then :)